physics_simulation.py

```diff
@@ -348,7 +349,22 @@ def get_error_rate(sender, receiver) -> float:
         and "x" in receiver.keys()
         and "y" in receiver.keys()
     )
-    return free_space_path_loss(sender["x"], sender["y"], receiver["x"], receiver["y"])
+
+    # Calculate error rate based on FSPL
+    error_rate = free_space_path_loss(sender["x"], sender["y"], receiver["x"], receiver["y"])
+
+    # Check if we're in 'high_error' mode
+    if 'high_error' in sys.argv:
```

Comment:
we usually call physics simulation as a library from network_sim.py. Would it be possible to make high_error a boolean that defaults to True as an optional parameter to get_stats?

Something like
```py
def get_stats(t: int, high_error: bool = True):
```